src/tools/rbd/action/DiskUsage.cc

```diff
@@ -180,9 +180,7 @@ static int do_disk_usage(librbd::RBD &rbd, librados::IoCtx &io_ctx,
           goto out;
         }
 
-        if (snapname != NULL) {
-          total_prov += snap->size;
-        }
+        total_prov += snap->size;
```

Comment:
The originally thinking was that when a user creates an image X w/ provisioned size Y -- the fact that they have one or more snapshots associated with the image doesn't change the original provisioned sized of the image. Instead, it highlights the "hidden" fact that snapshots are not free -- at least that was the original intent.